src/focus-visible.js

```diff
@@ -132,7 +132,6 @@ function init() {
 
     if (hadKeyboardEvent || focusTriggersKeyboardModality(e.target)) {
       addFocusVisibleClass(e.target);
-      hadKeyboardEvent = false;
```

Comment:
Why this change?

Reply:
In my testing, it seemed like the screen reader swallowed keydown events. Instead, focus would just move to an element, which would then hit the line above and set `hadKeyboardEvent` to false. Meaning the next time the screen reader moved focus, it would seem as if it came from a mouse.

Reply:
Blargh. I hate that GitHub now submits comments when I hit shift-enter.

Continuing from the above...

The polyfill currently starts with `hadKeyboardEvent` set to `true`. And if we ever detect any mouse event, we set it to `false` (https://github.com/WICG/focus-visible/blob/master/src/focus-visible.js#L116).

So removing this line makes the keydown-swalling-screen-reader scenario work. And it doesn't seem to break anything because `onPointerDown` should handle turning `hadKeyboardEvent` off if the user ever switches to mouse.

Reply:
Does this result in being permanently stuck in focus-visible mode once any key is pressed?

Reply:
Apologies for the crossed comment trajectories.

Makes sense - this would probably also mean that a keypress that causes a modal to open would also stay in focus-visible mode.